### ISSUE TITLE
FATAL: Error, plugins collide by name anti-bloat

### ISSUE THREAD

**Issue by intervisionlord** [Nuitka/Nuitka#1725]
* Nuitka version, full Python version, flavor, OS, etc. as output by *this*
  command.

  > python -m nuitka --version
1.0.1
```
python -m nuitka --version
FATAL: Error, plugins collide by name anti-bloat: <class 'AntiBloatPlugin.NuitkaPluginAntiBloat'> <-> (<class 'AntiBloat.NuitkaPluginAntiBloat'>, None)
```

* How did you install Nuitka and Python
via pip

* The specific PyPI names and versions

  It should be taken from this output if there specific packages involved, e.g.
  `numpy`, you are expected to shorten this to the relevant ones.

  > python -m pip freeze
```
agate==1.6.3
agate-dbf==0.2.2
agate-excel==0.2.5
agate-sql==0.5.8
altgraph==0.17.2
ansicon==1.89.0
appdirs==1.4.4
asgiref==3.4.1
astroid==2.9.0
asttokens==2.0.5
atomicwrites==1.4.0
attrs==21.2.0
Babel==2.9.1
backcall==0.2.0
beautifulsoup4==4.10.0
black==21.12b0
blessed==1.19.0
bs4==0.0.1
cachetools==4.2.4
certifi==2021.10.8
cffi==1.15.0
charset-normalizer==2.0.7
click==8.0.3
colorama==0.4.4
commonmark==0.9.1
coverage==6.2
coveralls==3.3.1
cryptography==36.0.1
csvkit==1.0.6
dbfread==2.0.7
decorator==5.1.1
Django==4.0.1
docopt==0.6.2
et-xmlfile==1.1.0
executing==0.8.2
flake8==4.0.1
flyingcircus==0.1.4.0
future==0.18.2
greenlet==1.1.2
idna==3.3
imageio==2.21.0
iniconfig==1.1.1
isodate==0.6.1
isort==5.10.1
jedi==0.18.1
jinxed==1.1.0
lazy-object-proxy==1.6.0
leather==0.3.4
Markdown==3.3.6
matplotlib-inline==0.1.3
mccabe==0.6.1
mido==1.2.10
mypy==0.931
mypy-extensions==0.4.3
mysql-connector-python==8.0.29
Nuitka==1.0.1
Nuitka64==6.1.186
numpy==1.23.1
olefile==0.46
openpyxl==3.0.9
packaging==21.3
pandas==1.3.5
parsedatetime==2.4
parso==0.8.3
pathspec==0.9.0
pefile==2021.9.3
pep8==1.7.1
pickleshare==0.7.5
Pillow==9.0.0
pip-search==0.0.7
platformdirs==2.4.0
pluggy==1.0.0
progress==1.6
prompt-toolkit==3.0.24
protobuf==4.21.2
psutil==5.9.1
pure-eval==0.2.1
py==1.11.0
py2exe==0.11.0.1
pyaml==21.10.1
pycodestyle==2.8.0
pycparser==2.21
pydocstyle==6.1.1
pydoctest==0.1.15
pyflakes==2.4.0
pygame==2.1.2
Pygments==2.10.0
pyinstaller==4.7
pyinstaller-hooks-contrib==2021.3
pylama==8.3.6
pylava==0.3.0
pylint==2.12.2
pyOpenSSL==21.0.0
pyparsing==3.0.6
PyQt5==5.15.7
PyQt5-Qt5==5.15.2
PyQt5-sip==12.11.0
pyTelegramBotAPI==4.2.0
pytest==6.2.5
python-dateutil==2.8.2
python-gettext==4.0
python-slugify==5.0.2
python-vlc==3.0.12118
pytimeparse==1.1.8
pytube==11.0.2
pytube3==9.6.4
pytz==2021.3
pywin32-ctypes==0.2.0
PyYAML==6.0
requests==2.26.0
rich==10.16.1
setuptools-scm==6.3.2
shell==1.0.1
six==1.16.0
snowballstemmer==2.2.0
soupsieve==2.3.1
SQLAlchemy==1.4.31
sqlparse==0.4.2
stack-data==0.1.3
termcolor==1.1.0
text-unidecode==1.3
toml==0.10.2
tomli==1.2.3
tqdm==4.62.3
traitlets==5.1.1
types-PyYAML==6.0.3
typing_extensions==4.0.1
tzdata==2021.5
urllib3==1.26.7
var-dump==1.2
wcwidth==0.2.5
wrapt==1.13.3
xlrd==2.0.1
```

**Comment by kayhayen** [comment]
That looks unhealthy

> Nuitka64==6.1.186

**Comment by hmargera** [comment]
I am also running on this fatal error.
I tried downgrading python, downgrading Nuitka, but the issue stays.
Do you have any idea where the problem is comming from ?

>python --version
Python 3.9.5

>python -m nuitka --version
FATAL: Error, plugins collide by name anti-bloat: <class 'AntiBloatPlugin.NuitkaPluginAntiBloat'> <-> (<class 'AntiBloat.NuitkaPluginAntiBloat'>, None)

>python -m pip freeze
...
Nuitka==1.4.7
Nuitka64==7.1.60
...

Thx

**Comment by kayhayen** [comment]
Remove the old Nuitka installation, it conflicts.
